docker-compose.dev.oracle.yml

```diff
@@ -21,33 +21,33 @@ services:
     container_name: postgres-homs
     restart: always
     environment:
-      PGDATA: /var/lib/postgresql/data/homs
+      PGDATA: $HOMS_DB_PATH
       POSTGRES_DB: $HOMS_DB_NAME
       POSTGRES_USER: $HOMS_DB_USER
       POSTGRES_PASSWORD: $HOMS_DB_PASSWORD
       TZ: Europe/Moscow
     volumes:
       - $HOMS_PATH/data/homs/postgresql:/var/lib/postgresql/data
```

Comment:
`:$HOMS_DB_PATH`.
Same with other db data volumes.

Reply:
Don't understand what is wrong.